internal/jobcontainers/jobcontainer.go

```diff
@@ -205,21 +207,23 @@ func (c *JobContainer) CreateProcess(ctx context.Context, config interface{}) (_
 		return nil, errors.Wrapf(err, "failed to get application name from commandline %q", conf.CommandLine)
 	}
 
-	var token windows.Token
-	if getUserTokenInheritAnnotation(c.spec.Annotations) {
-		token, err = openCurrentProcessToken()
-		if err != nil {
-			return nil, err
-		}
-	} else {
-		token, err = processToken(conf.User)
-		if err != nil {
-			return nil, errors.Wrap(err, "failed to create user process token")
+	// We've already done the user dance already, so this is most likely an exec being launched. Just use the token we
+	// already retrieved when we made the init process.
+	if c.token == 0 {
```

Comment:
Does `token == 0` mean we've already done the user dance, or that we have not created a token yet?

Reply:
Oh this comment is out of date, good catch. If it's not 0 we've already created a token. This should be something to the tune of:
"If we haven't grabbed a token yet this is the init process being launched. Skip grabbing another token afterwards if we've already done the work."

Reply:
Fixed the comment!